src/createServer.js

```diff
@@ -1,8 +1,33 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const { validateUrl } = require('./validateUrl');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const fullPath = validateUrl(req, res);
+
+    // eslint-disable-next-line no-console
+    if (!fullPath) {
+      return;
+    }
+
+    fs.readFile(fullPath, { encoding: 'utf8' }, (err, data) => {
+      res.setHeader('Content-Type', 'text/plain');
+
+      if (err) {
+        res.statusCode = 404;
```

Comment:
Move all status code in constants ( for all files)

Reply:
Please have a look. Is that what you have asked for?